src/basics.php

```diff
@@ -143,6 +143,18 @@ function dd($var, $showHtml = null): void
             'file' => $trace[0]['file'],
         ];
 
+       if (
+            PHP_SAPI !== 'cli'
+            && $_SERVER['HTTP_REFERER']
```

Comment:
There was no need to make this multiline.
Best to use an IDE, the commits are getting out of hand a bit here.

Reply:
Also, this does not meet the comment I gave, about using !empty() to avoid the undefined index notice.